enhancements/worker-latency-profile/worker-latency-profile.md

```diff
@@ -107,89 +102,13 @@ const (
-  // Manager or Kube API Server) is actively rolling out, propagating changes to the
-  // respective arguments.
-  WorkerLatencyProfileProgressing WorkerLatencyStatusConditionType = "Progressing"
-
-  // Complete indicates whether the component (Kubelet or Controller Manager or Kube API Server)
-  // is successfully updated the respective arguments.
-  WorkerLatencyProfileComplete WorkerLatencyStatusConditionType = "Complete"
-
-  // Degraded indicates that the component (Kubelet or Controller Manager or Kube API Server)
-  // does not reach the state 'Complete' over a period of time
-  // resulting in either a lower quality or absence of service.
-  // If the component enters in this state, "Default" WorkerLatencyProfileType
-  // rollout will be initiated to restore the respective default arguments of all
-  // components.
-  WorkerLatencyProfileDegraded WorkerLatencyStatusConditionType = "Degraded"
-)
-
-type WorkerLatencyStatusConditionOwner string
-
-const (
-  // Machine Config Operator will update condition status by setting this as owner
-  MachineConfigOperator WorkerLatencyStatusConditionOwner = "MachineConfigOperator"
-
-  // Kube Controller Manager Operator will update condition status  by setting this as owner
-  KubeControllerManagerOperator WorkerLatencyStatusConditionOwner = "KubeControllerManagerOperator"
-
-  // Kube API Server Operator will update condition status by setting this as owner
-  KubeAPIServerOperator WorkerLatencyStatusConditionOwner = "KubeAPIServerOperator"
-)
-
-type WorkerLatencyStatusCondition struct {
-	// Owner specifies the operator that is updating this condition
-	// +kubebuilder:validation:Required
-	// +required
-	Owner WorkerLatencyStatusConditionOwner string `json:"owner"`
-
-	// type specifies the aspect reported by this condition.
-	// +kubebuilder:validation:Required
-	// +required
-	Type WorkerLatencyStatusConditionType `json:"type"`
-
-	// status of the condition, one of True, False, Unknown.
-	// +kubebuilder:validation:Required
-	// +required
-	Status ConditionStatus `json:"status"`
-
-	// lastTransitionTime is the time of the last update to the current status property.
-	// +kubebuilder:validation:Required
-	// +required
-	LastTransitionTime metav1.Time `json:"lastTransitionTime"`
-
-	// reason is the CamelCase reason for the condition's current status.
-	// +optional
-	Reason string `json:"reason,omitempty"`
-
-	// message provides additional information about the current condition.
-	// This is only to be consumed by humans.  It may contain Line Feed
-	// characters (U+000A), which should be rendered as new lines.
-	// +optional
-	Message string `json:"message,omitempty"`
-}
 ```
 
+### Status Updates
+
+The progress of the WorkerLatencyProfile rollout can be tracked in the following ways.
+* The Machine Config Operator generates events incase of any rollout failures. The "worker" machine config pool gets updated incase of a successful WorkerLatencyProfile rollout.
```

Comment:
```suggestion
* The Machine Config Operator generates events in case of any rollout failures. The "worker" machine config pool gets updated in case of a successful WorkerLatencyProfile rollout.
```

Reply:
Thanks, Addressed